src/__tests__/__snapshots__/config.test.js.snap

```diff
@@ -179,7 +179,7 @@ Object {
   "maxObjSize": 450000,
   "npmDownloadsEndpoint": "https://api.npmjs.org/downloads",
   "npmRegistryEndpoint": "https://replicate.npmjs.com/registry",
-  "npmRootEndpoint": "https://api.npmjs.org",
+  "npmRootEndpoint": "https://registry.npmjs.org",
```

Comment:
doh!

Reply:
I didn't realise the error because it was supposed to check for 404 haha